liri.js

```diff
@@ -1,186 +1,157 @@
 // get input from user and set it as an array while ignoring first 2 index's
 var inputArr = process.argv.slice(3);
+var Twitter = require('twitter');
+var dataArr = [];
+var Spotify = require('node-spotify-api');
+var request = require('request');
+var fs = require('fs');
+var keys = require("./keys.js");
+
+// twitter function
+var twitterFunc = function() {
+	var twitter = new Twitter ({
+	  	consumer_key: keys.twitterKeys.consumer_key,
+	  	consumer_secret: keys.twitterKeys.consumer_secret,
+	  	access_token_key: keys.twitterKeys.access_token_key,
+	  	access_token_secret: keys.twitterKeys.access_token_secret
+	});
+	// parameters for twitter to select the dril account and limit to 10 posts 
+	var params = {
+		screen_name: 'dril',
+		count: 10
+	};
+
+	twitter.get('statuses/user_timeline', params, function(error, tweets, response) {
+	  	if (!error && response.statusCode === 200) {
+	  		for (var i=0; i<tweets.length; i++) {
+	    		console.log(tweets[i].text);
+	    		console.log('-----------------');
+	    		dataArr.push(tweets[i].text);
+	    	}	
+	  	}
+	  	else {
+	  		console.log("meep");
+	  	}
+	  	// append to random.txt
+	    appendFunc();
+	});
+}
+
+// spotify function
+var spotifyFunc = function() {
+	spotifySong = inputArr;
+	// if user doesn't input a song, default to who let the dogs out
+	if (spotifySong == '') {
+		spotifySong = 'Who let the dogs out?';
+	}
+	var spotify = new Spotify ({
+	  	id: keys.spotifyKeys.id,
+		secret: keys.spotifyKeys.secret
+	});
+	spotify.search ({ 
+		type: 'track', 
+		query: spotifySong
+	}, function(error, data) {
+		if (error) {
+	    	return console.log('Error occurred: ' + error);
+		}
+		else {
+			for (var i=0; i<data.tracks.items.length; i++) {
+				// artist
+				console.log(data.tracks.items[i].artists[0].name);
+				// name
+				console.log(data.tracks.items[i].name);
+				// preview link to song, there are many times that it returns null so I used an if func for that
+				if (data.tracks.items[i].preview_url === null) {
+					console.log("There is no preview url for this song unfortunately");
+				}
+				else {	
+					console.log(data.tracks.items[i].preview_url);
+				}	
+				// album song is from 
+				console.log(data.tracks.items[i].album.name);
+				// divider
+				console.log('---------------------');
+				dataArr.push(
+					data.tracks.items[i].artists[0].name, 
+					data.tracks.items[i].name,
+					data.tracks.items[i].preview_url,
+					data.tracks.items[i].album.name
+				);
+			}
+			appendFunc();
+		}	
+	});
+}
 
+// OMDB
+var OMDBfunc = function() { 
+	var movieName = '';
+	movieName = inputArr.join('+');
+	// if user doesn't input any movie, default to Trainspotting which is a great film
+	if (movieName == '') {
+		movieName = 'Trainspotting';
+	}
```

Comment:
this can just be re-written using a ternary operator:
```javascript
var movieName = inputArr.length < 1 ? "Trainspotting" : inputArr.join("+");
```